InfoMovies/InfoMovies/Views/BasePage.xaml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="InfoMovies.Views.BasePage"
+             NavigationPage.HasNavigationBar="false"
```

Comment:
How will the user navigate back from item details to the main page in iOS ?

Reply:
i'm really newbie at IOS development..
So, i dont tested with this plataform and for that reason i forget the creation of back buton

Im  thinkig about `swipe gesture` or 'back float button' to bring it back 
(in this project i didnt not want to use navigationbar for design of app)